feature2/hide.txt

```diff
@@ -0,0 +1 @@
+i want to hide you some thing 
```

Comment:
write clearly what you want to hide ?

Reply:
i have changed the file that i wanted to hide